tests/conftest.py

```diff
@@ -1,15 +1,16 @@
 """Pytest conftest."""
 # pylint: disable=redefined-outer-name
-
+import asyncio
 import pytest
 
 from kytos.lib.helpers import get_controller_mock, get_switch_mock
 from napps.kytos.of_core.main import Main
 
 
 @pytest.fixture(autouse=True)
-def ev_loop(monkeypatch, event_loop) -> None:
+def ev_loop(monkeypatch) -> None:
```

Comment:
Can you double check if this fixture can be entirely nuked? 

Back then we use to `autouse=True` this since `event_loop` fixture needed to be passed around when getting `asyncio.get_running_loop()` on async def test cases. But looks like in newer versions from `pytest-asyncio` that might not be needed.

Reply:
Nuked. Now the tests that used this are `async`